lib/ExpensiMark.js

```diff
@@ -346,6 +348,39 @@ export default class ExpensiMark {
         return replacedText;
     }
 
+    /**
+     * Map the block to newline
```

Comment:
Please add more details on what this function does. We know that it doesn't replace all the blocks. So please mention those cases.

Thanks!